main.sh

```diff
@@ -22,14 +24,20 @@ else
     echo "! Skipping Parser !"
 fi
 
-cd src
 if [ "$SKIP_BOT" = "false" ]; then
+    cd src
     echo "Running DeadBot..."
     python3 deadbot.py
+    cd ..
 else
     echo "! Skipping DeadBot !"
 fi
 
+# cleanup
+if [ "$OUTPUT_DIR/decompiled-data" ]; then
+    rm -rf $OUTPUT_DIR/decompiled-data
+fi
```

Comment:
Is it possible that when its ran via docker that it purges decompiled data, rather than every time main.sh is ran? I foresee that being helpful in the future.

Reply:
I added two things:
* env var `WORK_DIR` setting decompiled-data directory
* env var `CLEANUP` to enable cleanup if desired